### DISCUSSION TITLE
Font Size

### DISCUSSION THREAD

**Discussion by chazbz** [ImageMagick/ImageMagick#6868] [General]
I am using Linux Mint 20.3 Uma, Cinnamon. The Imagemagick program comes with the distribution. My 76 year old eyes, have trouble seeing the options available when I open the program. Is there away to increase the font size of those options?

**Comment by fmw42** [comment]
Font size in what tool or command line?  In command line, use -pointsize to change the size of the font?  In the GUI, I don't think there is a way other than in your monitor resolution.
